website/src/pages/showcase.js

```diff
@@ -59,56 +61,137 @@ const renderLinks = app => {
   ) : null;
 
   return (
-    <p>
+    <p className="showcaseLinks">
       {linkPlayStore}
-      {linkPlayStore && linkAppStore ? ' • ' : ''}
+      {linkPlayStore && linkAppStore ? <span> • </span> : ''}
       {linkAppStore}
     </p>
   );
 };
 
+const randomizeApps = apps =>
+  apps.filter(app => !app.group).sort(() => 0.5 - Math.random());
+
 const Showcase = () => {
   const {siteConfig} = useDocusaurusContext();
 
   const showcaseApps = siteConfig.customFields.users;
-  const pinnedApps = showcaseApps.filter(app => app.pinned);
-  const featuredApps = showcaseApps
-    .filter(app => !app.pinned)
-    .sort((a, b) => a.name.localeCompare(b.name));
-  const apps = pinnedApps.concat(featuredApps);
+  const [randomizedApps, setRandomizedApps] = useState([]);
+
+  useEffect(() => {
+    setRandomizedApps(randomizeApps(showcaseApps));
+  }, []);
+
+  const metaApps = showcaseApps.filter(app => app.group === 'Meta');
+  const microsoftApps = showcaseApps.filter(app => app.group === 'Microsoft');
+  const shopifyApps = showcaseApps.filter(app => app.group === 'Shopify');
```

Comment:
I like the idea of improving this code, however I don't want to keep JSX/HTML for the group description in JSON. I feel like it introduces the unnecessary complexity.

Same with logos, where I have tried to set unified `max-height` or `max-width`  to avoid setting it directly, but both approaches results in uneven logos.

Saying that I'm going to introduce the groupings but only for apps and refactor a code a bit.